zzzz_modular_occulus/code/game/objects/items/weapons/implant/implants/carrion/smoker_spider.dm

```diff
@@ -0,0 +1,25 @@
+/obj/item/weapon/implant/carrion_spider/smoker
+	name = "smoker spider"
+	desc = "A tiny spider with a pearl-white abdomen covered in smooth bubbles."
+	icon = 'zzzz_modular_occulus/icons/obj/carrion_spiders.dmi'
+	icon_state = "spiderling_smoker"
+	spider_price = 10
+	var/datum/effect/effect/system/smoke_spread/bad/smoke
+
+/obj/item/weapon/implant/carrion_spider/smoker/New()
+	..()
+	smoke = new
```

Comment:
The smoke effect ought to be explicitly qdel'd once the spiderling itself is qdel'd, but the GC should probably handle that automatically without lagging the server to hell. Probably. Just keep this in mind if things start lagging to hell next time there's a carrion round on the server.

Reply:
Does this new patch fix the issue that _Would_ have been?

Reply:
It should!